src/javascript/esutil.ts

```diff
@@ -157,12 +165,24 @@ export function toScannedPolymerProperty(
   let description =
       jsdoc.removeLeadingAsterisks(getAttachedComment(node) || '').trim();
 
+  const name = objectKeyToString(node.key);
+  if (name == null) {
```

Comment:
wait... `false != null`? How did I forget that???

Nevermind then, there's definitely value in distinguishing between the two